docs/migration-7.10-11.md

```diff
@@ -302,6 +305,85 @@ task copySimpleSMBWhiteStore(type: Copy) {
 zipShare.dependsOn copySimpleSMBWhiteStore
 ```
 
+### Wiring Files using the Configuration Framework
+
+Files in `cluster` and `domains` directories need to be wired using the Configuration Framework. The automated
+migration step moved these files to their new locations, but manual configuration is required:
+
+1. Cluster- and Domain-specific config (`/src/main/resources/resources/{cartridgeName}/config/cluster`,
+   `/src/main/resources/resources/{cartridgeName}/config/domains/{domainName}`): Requires wiring in `configuration.xml`
+
+2. Cartridge- (app-type-)specific config (`/src/main/resources/resources/{cartridgeName}/config`): No wiring in
+   `configuration.xml`
+
+Verify that the configuration is correctly loaded at runtime by checking the server logs during startup.
+For more details about the configuration framework, refer to the [Concept - Configuration](https://support.intershop.com/kb/index.php/Display/301L43)
+Guide in the ICM documentation.
+
+### Adapt Logback Configuration
+
+In ICM 11, especially in cloud environments, the logback configuration needs to be adjusted to prevent issues with
+multiple application servers trying to write to the same log file.
+
+1. Remove file appenders from your logback configuration files:
+   - Locate your logback configuration files in `src/main/resources/resources/{cartridgeName}/logback/`
+   - Remove any `<appender>` configurations that write to files
+   - For example, remove configurations like:
+       ```xml
+       <appender name="DEBUG_LOG" class="ch.qos.logback.core.rolling.RollingFileAppender">
+           <file><@loggingDir@>/debug.log</file>
+           <encoder>
+               <pattern>%d{yyyy-MM-dd HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
+           </encoder>
+           <rollingPolicy class="ch.qos.logback.core.rolling.TimeBasedRollingPolicy">
+               <FileNamePattern><@loggingDir@>/debug-%d{yyyy-MM-dd}.log.zip</FileNamePattern>
+           </rollingPolicy>
+       </appender>
+       ```
+
+2. Use console logging instead:
+   - Ensure you have a console appender configured:
+       ```xml
+       <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+           <encoder>
+               <pattern>%d{"yyyy-MM-dd'T'HH:mm:ss.SSSXXX", UTC} [%thread] %-5level %logger{36} %msg%n</pattern>
+           </encoder>
+       </appender>
+       ```
+
+   - Make sure the root logger or specific loggers use the console appender:
+       ```xml
+       <root level="INFO">
+             <appender-ref ref="STDOUT" />
+       </root>
+       ```
+
+These changes ensure that your application works correctly in ICM 11 cloud environments where multiple instances of the
+application server may be running simultaneously.
```

Comment:
is more or less the same sentence as in lines 325+. Please rephrase. How about pointing out the log aggregation in cloud environments?